src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1470,6 +1470,16 @@ void PolicyHandler::SetRemoteControl(bool enabled) {
   POLICY_LIB_CHECK_VOID();
   policy_manager_->SetRemoteControl(enabled);
 
+  OnCountryConsentChanged(enabled);
```

Comment:
SetRemoteControl is changing user_consent_passengersRC. I am not sure this call must be here.